src/DssSpell.sol

```diff
@@ -21,27 +21,36 @@ pragma solidity 0.6.12;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
-
-interface ERC20Like {
-    function approve(address, uint256) external returns (bool);
-}
+// import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
 interface RwaUrnLike {
-    function lock(uint256) external;
     function draw(uint256) external;
 }
 
-contract DssSpellAction is DssAction, DssSpellCollateralAction {
+interface VestLike {
+    function yank(uint256) external;
+    function restrict(uint256) external;
+    function create(
+        address usr,
+        uint256 tot,
+        uint256 bgn,
+        uint256 tau,
+        uint256 eta,
+        address mgr
+    ) external returns (uint256);
+}
+
+contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/969f04cfec25e56791fbe4503bcbe2df7a58df1e/governance/votes/Executive%20vote%20-%20July%2029%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO/governance/votes/Executive%20vote%20-%20August%203%2C%202022.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-07-29 MakerDAO Executive Spell | Hash: 0x18850080b101bb43125dd2bec32e5e7196586c5614bb9a0f05f9bbe392901d64";
-
-    address constant RWA_TOKEN_FAB = 0x2B3a4c18705e99bC29b22222dA7E10b643658552;
+        "2022-08-03 MakerDAO Executive Spell | Hash: TODO";
 
-    uint256 constant RWA009_DRAW_AMOUNT = 25_000_000 * WAD;
+    uint256 public constant WAD                = 10**18;
+    address public constant RWA009_A_URN       = 0x1818EE501cd28e01E058E7C283E178E9e04a1e79;
```

Comment:
This should come from chainlog.

Reply:
done

Reply:
Perfect, just a note, instead of using local variables, you could replace the prev constants by immutables and call directly the DssExecLib.getChangelog(...) inline, so basically does the same than putting the call in the constructor. It is also very helpful as that way you do not add to the stack too deep issue. Anyway here the spell uses so low amount of variables so both options are fine. One probably more costly on deployment and the other on execution.